lib/BC/SleighLifter.cpp

```diff
@@ -247,7 +249,8 @@ class SleighLifter::PcodeToLLVMEmitIntoBlock : public PcodeEmit {
   PcodeToLLVMEmitIntoBlock(llvm::BasicBlock *target_block,
                            llvm::Value *state_pointer, const Instruction &insn,
                            SleighLifter &insn_lifter_parent,
-                           std::vector<std::string> user_op_names_)
+                           std::vector<std::string> user_op_names_,
+                           llvm::BasicBlock *exit_block_)
```

Comment:
This is starting to have too much args, consider grouping them up in structures.